polaris-service/src/test/java/io/polaris/service/admin/PolarisServiceImplIntegrationTest.java

```diff
@@ -107,10 +107,9 @@ public void tearDown() {
       response.readEntity(Catalogs.class).getCatalogs().stream()
           .forEach(
               catalog -> {
```

Comment:
Actually, those can then also be replaced w/ an expression lambda

Reply:
done!